tests/unit/source_code/test_lsp_plugin.py

```diff
@@ -0,0 +1,50 @@
+import tempfile
+from unittest.mock import Mock
+
+import pytest
+
+from pylsp import uris  # type: ignore
+from pylsp.workspace import Document, Workspace  # type: ignore
+from pylsp.config.config import Config  # type: ignore
+
+from databricks.labs.ucx.source_code import lsp_plugin
+
+
+@pytest.fixture
+def workspace(tmp_path) -> Workspace:
+    uri = tmp_path.absolute().as_uri()
+    config = Config(uri, {}, 0, {})
+    ws = Workspace(uri, Mock(), config=config)
+    return ws
+
+
+def temp_document(doc_text, ws):
+    with tempfile.NamedTemporaryFile(mode="w", dir=ws.root_path, delete=False) as temp_file:
+        name = temp_file.name
+        temp_file.write(doc_text)
+    doc = Document(uris.from_fs_path(name), ws)
+    return name, doc
+
+
+def test_pylsp_lint(workspace):
+    code = 'sc.emptyRDD()'
```

Comment:
we may want to add one more test with 2-3 lines of code to make sure line offsets are working fine.